test/api/v01/output_test.rb

```diff
@@ -265,21 +265,21 @@ def test_returned_keys_json_result
         problem: VRP.lat_lon,
         solver_name: 'vroom',
         expected_route_keys: %w[vehicle_id original_vehicle_id activities total_travel_time total_distance total_time start_time end_time],
-        expected_activities_keys: %w[point_id travel_distance travel_time travel_value service_id original_service_id detail current_distance type],
+        expected_activities_keys: %w[point_id travel_distance travel_time travel_value begin_time end_time service_id original_service_id detail current_distance type departure_time],
         expected_unassigned_keys: %w[point_id service_id original_service_id detail type reason]
       },
       ortools: {
         problem: VRP.lat_lon_capacitated,
```

Comment:
We need to change the problem to make sure we call _ortools_ because _vroom_ output is already tested in previous case.